mlflow/langchain/__init__.py

```diff
@@ -132,9 +132,15 @@ def _infer_signature_from_input_example_for_lc_model(
 ):
     from mlflow.langchain.utils.chat import _ChatResponse
 
-    signature, prediction = _infer_signature_from_input_example(
+    result = _infer_signature_from_input_example(
```

Comment:
`_infer_signature_from_input_example` returns None when an error occurs and `MLFLOW_TESTING` is false.